packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {TypedPattern} from '@agoric/internal'
+ * @import {StorageNode} from '@agoric/internal/src/lib-chainStorage.js'
+ * @import {MakeRecorderKit} from '@agoric/zoe/src/contractSupport/recorder.js'
+ * @import {USDCProposalShapes, ShareWorth} from '../pool-share-math.js'
+ */
+
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import {
+  AmountShape,
+  PaymentShape,
+  RatioShape,
+} from '@agoric/ertp/src/typeGuards.js';
+import { depositToSeat } from '@agoric/zoe/src/contractSupport/zoeHelpers.js';
+import { SeatShape } from '@agoric/zoe/src/typeGuards.js';
+import { M } from '@endo/patterns';
+import {
+  makeRecorderTopic,
+  TopicsRecordShape,
+} from '@agoric/zoe/src/contractSupport/topics.js';
+import {
+  deposit as depositCalc,
+  makeParity,
+  withdraw as withdrawCalc,
+  withFees,
+} from '../pool-share-math.js';
+import { makeProposalShapes } from '../type-guards.js';
+
+/**
+ * @param {Zone} zone
+ * @param {ZCF} zcf
+ * @param {Record<'USDC', Brand<'nat'>>} brands
```

Comment:
the `brands` param reads like it's the source of all brands. since this is just for one brand please just pass it in as a `usdcBrand` param, Or if you really want the destructuring something like `externalBrands`.

Reply:
oops. yeah. left-over from when a `PoolShare` brand was passed in